README.md

```diff
@@ -763,6 +765,59 @@ We have a small collection of algorithms, one for every day of the month. Scroll
     - [Introduction to matrices](https://en.wikipedia.org/wiki/Matrix_(mathematics))
 
 
+## **December 29 - Cup and String Phone**
+  - **Problem :**
+    - Tharun is playing with his friends using a string phone. To make this interesting he asked his friends to join him. He asked them to bring cup of their own and he will bring the strings for all of them so that all can communicate with each other. Now he wants to find the number of strings required to communicate with his friends. Help him with a program that takes a number of friends as input and give the number of strings as output.
+
+    For example Tharun plays with 2 of his friends. Then he will bring 3 Strings.This is because 1 string is enough to connect 2 friends.
+
+    <img src="/src/docs/dec29-string-phone.jpg" width="450" height="300">
+
+  - **Examples :**
+  
+    1) Input : 7\
+       Output : 28\
+       Explanation : \
+            Tharun & his 6 friends come together to play this game. So to connect all of 7 cups, we need 28 strings.\
+
+    2) Input : 10\
+       Output : 55
+ 
+  - **Uses :**
+    - Triangular Series
+    - Networking between computers 
+    - No. of greetings exchanged among 'n' people
+    - No. of matches played by 'n' teams
+
+  - **Resources :**
+    - [Triangular Series](https://en.wikipedia.org/wiki/Triangular_number)
+
+
+## **December 30 - Number of Diagonals**
+  - **Problem :**
+    - There is a n-sided polygon. Find the number of [diagonals](https://en.wikipedia.org/wiki/Diagonal) that are possible. Do not count duplicates lines. Diagonals are Line segments that connect the non-adjacent vertices.
+
+  - **Example :**
```

Comment:
Put the smallest examples at the start. Explain how you get that output. We want to make it clear and obvious. So put the rectangle at the start. Say that a rectangle has two diagonals, or better, show a diagram. Same for the pentagon.